LiteFarmOrg/LiteFarm
# Lf 2507 detail tab & retire sensor

This PR adds functionality to view sensor information such as name, reading types, brand, and more. 

To test:
1.  If you don't already have any, upload and add sensor(s) to your farm
2. Once you have sensors appearing on your farm map, you should be able to click them and arrive at the details page for the sensor
3. Ensure all the information you see in the details page aligns with the information in your database
4. Click the "Retire" button at the bottom of the screen and click retire on the modal. You should see a snackbar popup saying the sensor was successfully retired.
5. You should be redirected to your farm map, only this time, without the sensor you just retired

**Additionally**, if the sensor you uploaded and retired was a legitimate Ensemble sensor, you should also make sure the sensor was successfully unclaimed on Ensembles end. 
You can do this by calling the following endpoint `api.esci.io/organizations/{{organization_uuid}}/devices/unclaim/`. 
Make sure to include the correct body and auth token:
`
{
    "esid": "YOUR SENSOR ESID"
}
`
